README.md

```diff
@@ -5,6 +5,6 @@ A distroless ROCK image for Vault.
 ## Usage
 
 ```console
-docker pull ghcr.io/canonical/vault:1.14.3
-docker run -it ghcr.io/canonical/vault:1.14.3
+docker pull ghcr.io/canonical/vault:<version>
```

Comment:
Same comment as above

Reply:
For the README.md I'm not sure though, I don't want to tell people to clone the project, and use yq just to know what version to input here. We may want to publish our rocks with the `latest` tag as well and have our README.md point to use this tag.

@ghislainbourgeois @DanielArndt What do you think?

Reply:
I like that idea, and I agree with your reasoning.

Makes sense to me. :+1:

Reply:
All right so fot now I'll simply make the change in the CONTRIBUTING.md and remove the one in README.md. I'll add the "latest" tag in a different PR.

Reply:
Done

Reply:
PR #7 contains the changes to publish with "latest" tag